src/Tizen.Wearable.CircularUI.Chart.Forms/PieChartView.cs

```diff
@@ -0,0 +1,69 @@
+﻿/*
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Flora License, Version 1.1 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://floralicense.org/license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using Xamarin.Forms;
+
+namespace Tizen.Wearable.CircularUI.Chart.Forms
+{
+    /// <summary>
+    /// The PieChartView is a circular chart that shows information as a pie divided into slices. 
+    /// </summary>
+    public class PieChartView : ChartView
+    {
+        /// <summary>
+        /// BindableProperty. Identifies the ValueIsVisible bindable property.
+        /// </summary>
+        public static readonly BindableProperty ValueIsVisibleProperty = BindableProperty.Create(nameof(ValueIsVisible), typeof(bool), typeof(PieChartView), true);
+
+        /// <summary>
+        /// BindableProperty. Identifies the ValueFormat bindable property.
+        /// </summary>
+        public static readonly BindableProperty ValueFormatProperty = BindableProperty.Create(nameof(ValueFormat), typeof(ValueFormat), typeof(PieChartView), ValueFormat.Percent);
+
+        /// <summary>
+        /// BindableProperty. Identifies the LegendPosition bindable property.
+        /// </summary>
+        public static readonly BindableProperty LegendPostionProperty = BindableProperty.Create(nameof(LegendPosition), typeof(LegendPosition), typeof(PieChartView), LegendPosition.Bottom);
+
+        /// <summary>
+        /// Gets or sets visibility of each slice value.
+        /// </summary>
+        public bool ValueIsVisible
+        {
+            get { return (bool)GetValue(ValueIsVisibleProperty); }
+            set { SetValue(ValueIsVisibleProperty, value); }
+        }
+
+
+        /// <summary>
+        /// Gets or sets format of each slice value.
+        /// </summary>
+        public ValueFormat ValueFormat
```

Comment:
How about provide c# string formatting?
or not use `format`, `format` naming expect string formatting

Reply:
How about to change to ValueType ?

Reply:
It's good

Reply:
removed this property.  and I added ValueLabelTextFormat instead of this property.